packages/vm/lib/runBlock.ts

```diff
@@ -54,13 +54,9 @@ export interface RunBlockResult {
 }
 
 /**
- * Receipt generated for a transaction
+ * Abstract interface with common transaction receipt fields
  */
-export interface TxReceipt {
-  /**
-   * Status of transaction, `1` if successful, `0` if an exception occured
-   */
-  status: 0 | 1
+interface TxReceipt {
```

Comment:
Yes, since this is now kind of like an "abstract" receipt and the extended receipts now representing concrete receipt instances. Should I re-add the  export?

Reply:
So I was wondering because `RunBlockResult.receipts` has the type `TxReceipt[]` so its kinda exposed to users. Maybe then in that case change the `RunBlockResult.receipts` type to `PreByzantiumTxReceipt|PostByzantiumTxReceipt[]`?

Reply:
Ok, have updated as suggested.